README.md

```diff
@@ -341,6 +341,20 @@ in the Installation Guide.
     powerful machines, you might even consider `-j16`, as building with
     more jobs than CPU cores can speed things up further.
 
+    Alternatively, the `MAKEFLAGS` environment variable can be used.
+    In this case, only provide the flag itself, for example
+    `export MAKEFLAGS="-j4"`.
+
+    Note that the compilation may nonetheless uses a different number of
+    threads, because sometimes `ninja` is used.
+    Unfortunately, [there is no way to control number of jobs `ninja` uses
+    from environment variables](https://github.com/ninja-build/ninja/issues/1482).
+    See also https://github.com/sagemath/sage/issues/38950.
+
+    If the [Meson build system](https://doc-release--sagemath.netlify.app/html/en/installation/meson)
+    is used, the number of jobs running in parallel passed to `meson compile` will be respected,
+    because everything are managed by `ninja`.
```

Comment:
I think the only think that uses ninja is meson. Ninja does a good job at figuring out how parallel to build for most users. I am not sure it's necessary to comment on the details here. Maybe it's enough to say that the above only affects some parts of the build system and that it is not universally applied everywhere yet?

Maybe we could recommend to set `SAGE_NUM_THREADS` as well? I think it's used by things such as the docbuild but I have not double checked that this is (still) the case.